bazel/external/cargo/remote/ahash-0.3.8.BUILD

```diff
@@ -0,0 +1,46 @@
+"""
```

Comment:
Is the plan to check all of this into the main Envoy repo? Seems kind of terrifying from a maintenance perspective, even if automatically generated.

Reply:
Exactly. But maintaining these automatically generated is not inevitable. I think that we should force to use [this](https://github.com/envoyproxy/envoy-wasm/pull/613/files#diff-fefbda479b8fc667205a556beec93b64R1). And warn to use this script by DO NOT EDIT comment by inserting that "We must use tools/cargo-raze.sh".

Reply:
or, automatically generate with cargo-raze.sh from pre-push script by checking the Cargo.toml had staged or not.

Reply:
Meanwhile, yes, eventually I think it can made into rules_rust to generate WORKSPACE rule, just like rules_python's `pip3_import`.

Reply:
Definitely want a tracking issue for this one, since this is a lot of boilerplate that we should be able to minimize. But I'd be willing to take it as MVP.